config/ignition_citadel_gazebo11_ubuntu_focal.yaml

```
@@ -12,12 +12,25 @@ filter_formula: "\
  Package (= libgazebo11) |\
  Package (= libgazebo11-dev) \
 ), $Version (% 11.10.2-1*) |\
```

According to the generator script there is a new version of gazebo11 to import

```suggestion
), $Version (% 11.11.0-1*) |\
```